config/environment

```diff
@@ -17,10 +17,13 @@ fi
 
 AWS_DEFAULT_REGION='us-east-1'
 LAMBDA_DRIVER_FUNCTION_NAME="dcp-matrix-service-driver-${DEPLOYMENT_STAGE}"
+LAMBDA_NOTIFICATIONS_FUNCTION_NAME="dcp-matrix-service-notifications-${DEPLOYMENT_STAGE}"
 DYNAMO_STATE_TABLE_NAME="dcp-matrix-service-state-table-${DEPLOYMENT_STAGE}"
 DYNAMO_OUTPUT_TABLE_NAME="dcp-matrix-service-output-table-${DEPLOYMENT_STAGE}"
 MATRIX_RESULTS_BUCKET="dcp-matrix-service-results-${DEPLOYMENT_STAGE}"
 MATRIX_QUERY_BUCKET="dcp-matrix-service-queries-${DEPLOYMENT_STAGE}"
+MATRIX_PRELOAD_BUCKET="dcp-matrix-service-preload-${DEPLOYMENT_STAGE}"
+MATRIX_REDSHIFT_IAM_ROLE_ARN="arn:aws:iam::${ACCOUNT_ID}:role/matrix-service-redshift-${DEPLOYMENT_STAGE}"
```

Comment:
Why does this need to be in the environment file? I would think it just needs to exist in terraform?

Reply:
Redshift [COPY command](https://docs.aws.amazon.com/redshift/latest/dg/r_COPY.html#r_COPY-syntax-overview) requires an argument that specifies an IAM role to use.